package CrxFile;

message AsymmetricKeyProof {
    optional bytes PublicKey = 1;
    optional bytes Signature = 2;
}

message SignedData {
    optional bytes CrxId = 1;
}

message CrxFileHeader {
    repeated AsymmetricKeyProof Sha256WithRsa = 2;
    repeated AsymmetricKeyProof Sha256WithEcdsa = 3;
    optional bytes SignedHeaderData = 10000;
}

